crates/types/src/config/admin.rs

```diff
@@ -49,10 +49,10 @@ pub struct AdminOptions {
     /// # Log trim interval
     ///
     /// Controls the interval at which cluster controller tries to trim the logs. Log trimming
-    /// can be disabled by setting it to "".
-    #[serde(with = "serde_with::As::<Option<serde_with::DisplayFromStr>>")]
```

Comment:
I think that I forgot to put `#[serde_as(as = "serde_with::NoneAsEmptyString")]` to support `""` to be translated into `None`. "0s" is probably better.